macaroonbakery/json_serializer.py

```diff
@@ -70,7 +71,7 @@ def raw_b64decode(s):
 
 
 def caveat_to_dict(c):
-    ''' Caveat to dictionnary for the JSON macaroon V1 format.
+    '''Caveat to dictionnary for the JSON macaroon V1 format.
```

Comment:
I'm not sure this should be exported.

    ''' Return a caveat as a dictionary for export as the JSON
    macaroon v1 format
    '''

?

It would probably be good for the name to include the version number too.